GeneralSQL/Error.py,sha256=kBrYZ17K0Jps9-DY3QFG_tLa9GOrLBJvoTLv8wwxfMk,2321
GeneralSQL/__init__.py,sha256=pIojl_P1XFYx65uQkkpsL0kmRSGU74SNZi5zApuc3ZA,2155
GeneralSQL/Template/__init__.py,sha256=F80meUoOqQIOiYxE9ryCJzTMqFPBOwajkaf6Gk1e7Zk,1431
GeneralSQL/Template/base.py,sha256=oQTj7lN5JmlD3l3fG3-pm3dD8lPd0e6T1H5b0yOxToE,3024
GeneralSQL/db/Jmysql.py,sha256=7j5JBegU1g1eUGCukOWN4VO2ckwMjWNp42mYpft9gVQ,15534
GeneralSQL/db/__init__.py,sha256=PgWm-6m5ZDVt2vjNOtrgO7sQYzXa6A0p8Y2RnoZNTl0,57
GeneralSQL-1.1.0.dist-info/LICENSE,sha256=OXLcl0T2SZ8Pmy2_dmlvKuetivmyPd5m1q-Gyd-zaYY,35149
GeneralSQL-1.1.0.dist-info/METADATA,sha256=o1-88cDvLkP_kuj_uogA4HPLMrbaVpw-tIa-4JO1URU,1888
GeneralSQL-1.1.0.dist-info/WHEEL,sha256=oiQVh_5PnQM0E3gPdiz09WCNmwiHDMaGer_elqB3coM,92
GeneralSQL-1.1.0.dist-info/top_level.txt,sha256=N28y8Tp5KNt4vFIdlg6UXzrY9QGtXwgK-oqRvdRBuh8,11
GeneralSQL-1.1.0.dist-info/RECORD,,
